starter_code/app.js

```diff
@@ -1,18 +1,31 @@
 
-const express = require('express');
-const hbs     = require('hbs');
-const app     = express();
-const path    = require('path');
-const PunkAPIWrapper = require('punkapi-javascript-wrapper');
-const punkAPI = new PunkAPIWrapper();
+const express = require('express')
+const hbs = require('hbs')
+const app = express()
+const path = require('path')
+const PunkAPIWrapper = require('punkapi-javascript-wrapper')
+const punkAPI = new PunkAPIWrapper()
 
-app.set('view engine', 'hbs');
-app.set('views', __dirname + '/views');
-app.use(express.static(path.join(__dirname, 'public')));
+app.set('view engine', 'hbs')
+app.set('views', path.join(__dirname, '/views'))
+app.use(express.static(path.join(__dirname, 'public')))
 
 app.get('/', (req, res, next) => {
-  res.render('index');
-});
+  res.render('index')
+})
 
+app.get('/beers', (req, res, next) => {
+  const beers = punkAPI.getBeers()
+  beers.then(beer => {
+    res.render('beers')
```

Comment:
you have to pass data here, and also need a bit of refactoring (like the ```beers``` in line 19):
```
app.get('/beers', (req, res, next) => {
  punkAPI.getBeers()
  .then(beers => {
    res.render('beers', { beers });
  })
  .catch(error => console.log(error))
});
```